components/typography/stories/template.js

```diff
@@ -106,7 +112,413 @@ export const Template = (args = {}, context = {}) => {
 	/** Wrap items with the spectrum-Typography wrapper if there are more than 1 items (this ensures correct margins) */
 	return html`${when(
 		content.length > 1,
-		() => html`<div class="spectrum-Typography" id=${ifDefined(id)}>${processedContent}</div>`,
+		() => html`<div class="spectrum-Typography" id=${ifDefined(id)} lang=${lang || nothing}>${processedContent}</div>`,
```

Comment:
This was the most minimal way I could think of to add the lang attribute if it's passed

Reply:
Nice, I didn't know `nothing` would also work like that with attributes. `lang=${ifDefined(lang)}` might also work here if it doesn't need to also check against an empty string.